src/toggle/toggle.tsx

```diff
@@ -122,6 +185,177 @@ export const Toggle = ({
         );
     };
 
+    const renderCompositeOptionSection = () => {
+        if (!compositeOptionSection) {
```

Comment:
already did the check in the main `renderCompositeSection` so no need to check again here